open_buildings/cli.py

```diff
@@ -74,7 +75,7 @@ def get_buildings(geojson_input, dst, source, country_iso, silent, overwrite, ve
     else:
```

Comment:
`--geocode` can't be a flag. a flag is a boolean - either true or false. we want geocode to be a string input, similar to `--country_iso`